### ISSUE TITLE
Multiple Gearboxes Per Tractor?

### ISSUE THREAD

**Issue by rubberburner89** [Mogli12/GearboxAddon#350]
Hey Mogli12,

I'm currently working on editing tractor right now and have added multiple engine configurations and that got me to thinking... Is it possible to add multiple Gearboxes the same way? The tractor I'm working on did actually come with a couple different options and I'd like to implement that if possible. I've tried on my own to get this to work  but I keep hitting a wall. Thanks in advance,

Rubberburner89

**Comment by Mogli12** [comment]
Hallo Rubberburner89,

Ja, das geht. Du kannst in der XML vom Fahrzeug auf oberster Ebene die beiden Tags `transmissions`und `transmission` anstelle von `gearboxMogli` verwenden. Die Attribute kommen alle zu `transmission`. Beispiel:

```xml
...
  <motorConfigurations>
...
  </motorConfiguration>

  <transmissions>
    <transmission name="PowerShift" maxForwardSpeed="52" maxBackwardSpeed="32" ptoRpm="1900" ptoRpmEco="1600">
...
    </transmission>
  </transmissions>
```

Viele Grüße

**Comment by rubberburner89** [comment]
Yes, that worked. I am trying to add a price for the second transmission though, and having on the same line as the name did not work.

**Comment by Mogli12** [comment]
Hi,

Sorry, but pricing is not supported. The other options would be to have a `motorConfiguration` entry for every motor/gearbox combination. This would allow you to change decals and prices.

Regards,
Stefan

**Comment by rubberburner89** [comment]
Alright... thanks for your time, and an amazing mod :)

**Comment by Mogli12** [comment]
Let me think about the pricing...

**Comment by rubberburner89** [comment]
:)

**Comment by Mogli12** [comment]
Hi, 
with v3.1 build 78 the gearbox addon support the two attribute price and dailyUpKeep at the transmission tag. `<transmission><transmission>...</transmission></transmissions>` should now work in the vehicle.xml but only at top level, not below the `<motorConfigurations><motorConfiguration>...`.
Regards,
Stefan
